[Tablet Orders] Enable side by side orders in dev builds

joshheald opened the pull request:
<!-- Remember about a good descriptive title. -->

<!-- Id number of the GitHub issue this PR addresses. -->

## Description
<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->

Mostly tweaks were required to UI tests to have them continue working with the side-by-side view.

I wasn't hugely pleased with the `swipeUp` call I had to add to fix one of the tests, as it seems a bit hacked in. If anyone on @woocommerce/mobile-ui-testing-squad has any thoughts on how to better get `isFullyVisibleOnScreen()` to return false for the Add Note button when it's hidden below the summary totals drawer, that would be appreciated. See f9fd029

The `Cancel` button on the order form has been refactored a little to make it easier to set an accessibility identifier, so checking that it shows and works in both types of view may be helpful.

## Testing instructions
<!-- Step by step testing instructions. When necessary break out individual scenarios that need testing, consider including a checklist for the reviewer to go through. -->

Check UI tests pass.


---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

joshheald committed:
Apply orders accessibility ID to dismiss button

joshheald committed:
Enable side-by-side orders in dev builds

dangermattic commented:
<!--
  0 Errors
  0 Warnings
  1 Message: This PR is still a Draft: some...
  0 Markdowns
-->
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

joshheald committed:
Fix test_create_new_order on iPad in side-by-side

joshheald committed:
Fix test_add_note_to_existing_order on iPad

The `scrollintoView` function doesn’t perform any scrolling in this case because `isFullyVisibleOnScreen` is returning true, even when the note button is hidden (just) below the expandable totals drawer.

joshheald committed:
Add comment to explain scroll call

iamgabrielma committed:
lint: remove whitespace

iamgabrielma reviewed (approved):
LGTM. Just pushed a commit to fix a CI lint warning 👍
